instaclustr/resource_cluster.go

```diff
@@ -1143,6 +1152,13 @@ func doClusterResize(client APIClientInterface, clusterID string, d resourceData
 		} else {
 			return nil
 		}
+	case "CADENCE":
+		if hasCadenceSizeChanges(d) {
```

Comment:
Looks like `hasCassandraSizeChanges`, `hasRedisSizeChanges` and `hasCadenceSizeChanges` do the exact same thing. How about we merge them up into a single method? Example - `hasSimpleNodeSizeChanges`

Reply:
Good idea, on it.